Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -243,8 +243,25 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
     SSBLookupContext *context = [SSBLookupContext sharedLookupContext];
     if (!context)
         return listener.didReceiveSafeBrowsingResults({ });
-    [context lookUpURL:url completionHandler:makeBlockPtr([listener = Ref { listener }, forMainFrameNavigation, url = url] (SSBLookupResult *result, NSError *error) mutable {
-        RunLoop::protectedMain()->dispatch([listener = WTFMove(listener), result = retainPtr(result), error = retainPtr(error), forMainFrameNavigation, url = WTFMove(url)] {
+
+    // This gets adjusted whenever we get the response.
+    auto initialTimeout = Seconds(5_s);
```

Comment:
We don't want this.  Just start the timer when we receive the response if it hasn't received SSB results already.